src/mon/FSCommands.cc

```diff
@@ -295,16 +295,17 @@ class SetHandler : public FileSystemCommandHandler
         });
       }
     } else if (var == "balancer") {
-      if (val.empty())
+      if (val.empty()) {
         ss << "unsetting the metadata load balancer";
-      else 
+      } else {
         ss << "setting the metadata load balancer to " << val;
         fsmap.modify_filesystem(
             fs->fscid,
             [val](std::shared_ptr<Filesystem> fs)
         {
           fs->mds_map.set_balancer(val);
         });
+      }
```

Comment:
@liewegas I think this brace needs to go below line 301, not line 308. Then fsmap.modify_filesystem can set balancer to a string if val is not empty OR set balancer to empty if val is empty.